### ISSUE TITLE
iOS 14.2 AudioPlayer will not play: [The operation could not be completed]

### ISSUE THREAD

**Issue by ooglek** [rxlabz/audioplayer#145]
I am using a pre-signed HTTPS Amazon AWS S3 URL for the audio stream. On iOS 13 and using audioplayer 0.5.2 I had no issues with my current code.

I've tried replacing the pre-signed HTTPS AWS S3 URL with a static one: `https://luan.xyz/files/audio/ambient_c_motion.mp3` 
but still the player does not play.

On iOS 14 and audioplayer 0.8.1 the audio will not play, and the `audioPlayer.onPlayerStateChanged.listen()` throws an error of `[The operation could not be completed]`

Is this being actively developed or should I jump ship?

iOS 14.2
flutter 1.22.4
Xcode 12.2
dart 2.10.3
audioplayer 0.8.1

**Comment by ooglek** [comment]
Just as a test I downloaded the code from master and ran the `example/` program on IOS 14.2 with the same results -- no audio playing. Slider does not move. Audio volume on the simulator is turned up and the Silence switch is turned off (not orange).

I also deployed the `example/` Flutter program on IOS 13.7 on a iPhone 11 Pro and everything worked as expected.

As of now, **AUDIOPLAYER DOES NOT WORK ON IOS 14** as far as I can tell.
